.gradletasknamecache

```diff
@@ -0,0 +1,162 @@
+core:assemble
```

Comment:
I'd ignore it. Looks like a per-user, not per-project, configuration file

Reply:
Yes, I forgot add it to the gitignore.